components/portal/src/components/appLayout/header.js

```diff
@@ -0,0 +1,199 @@
+/*
```

Comment:
The file name should start with a capital letter if it contains a component. Please fix in all the files.